app/models/member.rb

```diff
@@ -36,6 +36,15 @@ class Member < ActiveRecord::Base
   scope :recently_joined, -> { reorder("confirmed_at desc") }
   scope :wants_newsletter, -> { where(newsletter: true) }
 
+  scope :interesting, lambda {
+    confirmed
+      .located
+      .recently_signed_in
+      .has_plantings
+  }
+
+  scope :has_plantings, -> { joins(:plantings).where("plantings.id IS NOT NULL") }
```

Comment:
I slightly prefer the `where.not(plantings: { id: nil }) }` style, but no biggie.